docs/overview/trustbloc-did-method.md

```diff
@@ -1,19 +1,22 @@
-# TrustBloc DID Method Specification
+# TrustBloc DID Method Specification 0.1
 
-## `did:trustbloc` Method Specification
-
-### 
+#### Spec Version
+This is version `0.1` of the Trustbloc DID Method Specification.
 
+## `did:trustbloc` Method Specification
 The `did:trustbloc` DID method allows groups of independent organizations to share custody of a DID registry on a permissioned ledger.
 
-Independent *stakeholder* organizations which wish to transact with one another using DIDs can come together to form a *consortium* to manage their shared custody of a ledger.
+Independent [*stakeholder*](#stakeholder-organization) organizations wishing to transact with one another using DIDs can come together to form a [*consortium*](#consortium) to manage their shared custody of a ledger.
 
-The DID method operates as a discovery service that allows resolvers to discover the stakeholder organizations within a consortium, verify that the consortium itself is endorsed by the stakeholders, and access sidetree endpoints owned and exposed by the individual stakeholders, which the resolver can subsequently use to perform DID document operations within the consortium.
+This spec defines a [discovery service](#discovery-service). The discovery service provided by did:bloc allows a resolver to verify that a consortium is endorsed by its constituent stakeholder organizations, verify that the configuration files of each stakeholder (which includes a list of sidetree endpoints) are signed by the respective stakeholders, and use the provided sidetree endpoints to perform sidetree DID operations.
 
-Definitions:
-
-- Consortium: a group of Stakeholder organizations which all manage DIDs with their own Sidetree nodes, all on the same ledger. A Consortium hosts the configuration files which a resolver uses when first connecting to the network.
-- Stakeholder organization (or just stakeholder): an independent organization which follows some cooperative rules with the other stakeholder organizations within the same consortium to allow them all to manage DIDs using their own Sidetree nodes on the same ledger, and allow resolvers to manage DIDs through any of the stakeholders. Stakeholders host ledger nodes, host sidetree endpoints, and sign policy changes.
+##### Definitions
+###### Discovery Service:
+A mechanism that allows resolvers to discover and validate DID operation endpoints.
+###### Consortium:
+A group of Stakeholder organizations which all manage DIDs with their own Sidetree nodes, all on the same ledger. A Consortium hosts the configuration files which a resolver uses when first connecting to the network.
+###### Stakeholder organization:
```

Comment:
why does it have to be an "organization"? why not a peer? Or just a "stakeholder"?

Reply:
can't Alice running a couple of sidetree nodes be a member of one of these consortiums?

Reply:
Good point.